lib/elix/brain/store.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Elix.Brain.Store do
+  @callback start_link :: any
```

Comment:
`@callback` is part of definining behaviours ([docs](http://elixir-lang.org/getting-started/typespecs-and-behaviours.html#defining-behaviours)). Here I'm saying that any module that implements `@behaviour Elix.Brain.Store` (i.e. the redis and process stores) must implement a `start_link` function with the signature declared here. If it doesn't, we get a compiler warning, so I've found it a handy way to gently enforce a consistent API when using polymorphism. Though I do find `@callback` to be a confusing name for this.